### DISCUSSION TITLE
(Improvement) smaller time increments

### DISCUSSION THREAD

**Discussion by Jorkim** [cpvalente/ontime#1120] [Q&A]
Hello,
at first thanks for the fast fix on my last problem.
Now i have a new possible feature or improvement.
Would it be possible to have smaller time increments than a second? I am using OnTime as a master software to control laser/lights/smoke machines etc... and camera angles and sounds.
For that i would like to see a smaller time increment to controll my cameras more precise.

Thanks for your help

**Comment by cpvalente** [comment]
Thank you for your suggestion @Jorkim 

If you are able to build the app yourself, you can configure this in this file
https://github.com/cpvalente/ontime/blob/master/apps/server/src/config/config.ts

you likely would only need to modify `notificationRate`

As for exposing this config to general app, we would need to think about what consequences that might come with. It is unlikely that we would that in the near future

Hope this helps
